training/common/Makefile.common

```diff
@@ -113,7 +114,7 @@ driver-toolkit:
 		$(ARCH:%=--platform linux/%) \
 		$(BUILD_ARG_FILE:%=--build-arg-file=%) \
 		$(ENABLE_RT:%=--build-arg ENABLE_RC=%) \
-		$(FROM:%=--from=%) \
+		$(DRIVER_TOOLKIT_BASE_IMAGE:%=--build-arg BASE_IMAGE=%) \
```

Comment:
Let's use `BASEIMAGE` to mantain coherence with other Containerfiles

Reply:
I reverted. But we're using `_` to separate words in environment variables, so it looked more consistent with the other variables. Maybe, I'll propose another PR to use `BASE_IMAGE` everywhere 😉